bcda/models/models_test.go

```diff
@@ -1042,3 +1042,163 @@ func (s *ModelsTestSuite) TestGetBlueButtonID_SuppressionMBI() {
+	}
+	defer s.db.Unscoped().Delete(&aco)
+
+	cclfFile := CCLFFile{CCLFNum: 8, ACOCMSID: acoCMSID, ImportStatus: constants.ImportComplete}
+	err = s.db.Save(&cclfFile).Error
+	if err != nil {
+		s.FailNow("Failed to save CCLF file", err.Error())
+	}
+	defer s.db.Unscoped().Delete(&cclfFile)
+
+	// Beneficiary 1
+	bene1 := CCLFBeneficiary{FileID: cclfFile.ID, BlueButtonID: "bene1_bbID"}
+	err = s.db.Save(&bene1).Error
+	if err != nil {
+		s.FailNow("Failed to save beneficiary", err.Error())
+	}
+	defer s.db.Unscoped().Delete(&bene1)
+
+	// Beneficiary 2
+	bene2 := CCLFBeneficiary{FileID: cclfFile.ID, BlueButtonID: "bene2_bbID"}
+	err = s.db.Save(&bene2).Error
+	if err != nil {
+		s.FailNow("Failed to save beneficiary", err.Error())
+	}
+	defer s.db.Unscoped().Delete(&bene2)
+
+	// Beneficiary 3
+	bene3 := CCLFBeneficiary{FileID: cclfFile.ID, BlueButtonID: "bene3_bbID"}
+	err = s.db.Save(&bene3).Error
+	if err != nil {
+		s.FailNow("Failed to save beneficiary", err.Error())
+	}
+	defer s.db.Unscoped().Delete(&bene3)
+
+	// Beneficiary 4
+	bene4 := CCLFBeneficiary{FileID: cclfFile.ID, BlueButtonID: "bene4_bbID"}
+	err = s.db.Save(&bene4).Error
+	if err != nil {
+		s.FailNow("Failed to save beneficiary", err.Error())
+	}
+	defer s.db.Unscoped().Delete(&bene4)
+
+	// Suppressed bene from 59 days ago. Will be included in GetSuppressedBlueButtonIDs result
+	bene1Suppression := Suppression{BlueButtonID: "bene1_bbID", PrefIndicator: "N", EffectiveDt: time.Now().Add(-(24 * 59) * time.Hour)}
+	err = s.db.Save(&bene1Suppression).Error
+	if err != nil {
+		s.FailNow("Failed to save suppression", err.Error())
+	}
+	defer s.db.Unscoped().Delete(&bene1Suppression)
+
+	// Suppressed bene from 59 days and 23 ago. Will be included in GetSuppressedBlueButtonIDs result
+	bene2Suppression := Suppression{BlueButtonID: "bene2_bbID", PrefIndicator: "N", EffectiveDt: time.Now().Add(-(24*59 + 23) * time.Hour)}
+	err = s.db.Save(&bene2Suppression).Error
+	if err != nil {
+		s.FailNow("Failed to save suppression", err.Error())
+	}
+	defer s.db.Unscoped().Delete(&bene2Suppression)
+
+	// Suppressed bene from 60 days ago. Will be included in GetSuppressedBlueButtonIDs result
+	bene3Suppression := Suppression{BlueButtonID: "bene3_bbID", PrefIndicator: "N", EffectiveDt: time.Now().Add(-(24 * 60) * time.Hour)}
+	err = s.db.Save(&bene3Suppression).Error
+	if err != nil {
+		s.FailNow("Failed to save suppression", err.Error())
+	}
+	defer s.db.Unscoped().Delete(&bene3Suppression)
+
+	// Suppressed bene from 59 days ago. Will be included in GetSuppressedBlueButtonIDs result
+	bene4Suppression := Suppression{BlueButtonID: "bene4_bbID", PrefIndicator: "N", EffectiveDt: time.Now().Add(-(24 * 61) * time.Hour)}
+	err = s.db.Save(&bene4Suppression).Error
+	if err != nil {
+		s.FailNow("Failed to save suppression", err.Error())
+	}
+	defer s.db.Unscoped().Delete(&bene4Suppression)
+
+	result := GetSuppressedBlueButtonIDs(db)
+	assert.Len(s.T(), result, 2)
```

Comment:
Can we also, just to safe, assert that bene3 and bene4 are not included?

Reply:
@PatKelsh is this comment still relevant?  i didn't see this get addressed, but maybe I'm missing the intent?